shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IAppInsightsCore.ts

```diff
@@ -80,4 +90,10 @@ export interface IAppInsightsCore extends IPerfManagerProvider {
      * Return a new instance of the IProcessTelemetryContext for processing events
      */
     getProcessTelContext() : IProcessTelemetryContext;
+
+    /**
+     * Find and return the (first) plugin with the specified identifier if present
+     * @param pluginIdentifier
+     */
+    getPlugin<T extends IPlugin = IPlugin>(pluginIdentifier: string): ILoadedPlugin<T>;
```

Comment:
Creating a unified "lookup" for a plugin, this happens throughout the code by lookup at the passed config. But as we are going to be updating the loaded extensions that's going to get problematic in later PR's